lib/resources/postgres.rb

```diff
@@ -36,6 +36,9 @@ def initialize
         version = if inspec.directory('/var/lib/pgsql/data').exist?
                     warn 'Found /var/lib/pgsql/data. Assuming postgresql install uses un-versioned directories.'
                     nil
+                  elsif inspec.directory('/var/lib/postgresql/data').exist?
```

Comment:
@kidbrax @chris-rock @adamleff I folded this into PR #1758 so given the updates we made to the resource in general.